mirascope/chat/utils.py

```diff
@@ -40,8 +55,20 @@ def patch_openai_kwargs(
     kwargs: dict[str, Any],
     prompt: Optional[Union[Prompt, str]],
     tools: Optional[list[Type[OpenAITool]]],
-):
-    """Sets up the kwargs for an OpenAI API call."""
+) -> None:
+    """Sets up the kwargs for an OpenAI API call.
+
+    Messages are parsed to have the required `role` and `content` items, and tools
+    (if any exist) are parsed into JSON schemas in order to fit the OpenAI API.
```

Comment:
make sure it's clear here that other kwargs remain untouched and that we only explicitly change messages and tools and that we won't overwrite tool_choice if provided etc.